src/EditView.vala

```diff
@@ -212,6 +212,9 @@ class EditView: Gtk.DrawingArea, Gtk.Scrollable {
 					core_connection.send_gesture(view_id, line, column, "toggle_sel");
 				} else {
 					core_connection.send_click(view_id, line, column, 0, 1);
+					if (event.button == 2) {
```

Comment:
Is there no constant in Gtk that can be used instead of the literal `2`?

Reply:
AFAIK there isn't. From the [documentation](https://developer.gnome.org/gdk3/stable/gdk3-Event-Structures.html#GdkEventButton):

> **guint button:** the button which was pressed or released, numbered from 1 to 5. Normally button 1 is the left mouse button, 2 is the middle button, and 3 is the right button. On 2-button mice, the middle button can often be simulated by pressing both mouse buttons together.

Reply:
I found it. It's `Gdk.BUTTON_MIDDLE`. Please use that instead of the literal `2`.